src/org/ods/component/Pipeline.groovy

```diff
@@ -207,6 +207,7 @@ class Pipeline implements Serializable {
                 containers: config.podContainers,
                 volumes: config.podVolumes,
                 serviceAccount: config.podServiceAccount,
+                slaveConnectTimeout: 15
```

Comment:
Is that minutes or seconds ? If seconds, I think it's too short, if minutes, too long I'd say 😄 10 minutes would still be plenty I think.

Reply:
thats seconds. how about a min?

Reply:
OK. I think I need to understand the meaning of the setting better then. When does the timeout start? When creating the pod, or when the pod is running? If it starts when the pod is created, it needs to be longer than one min I think. If it starts once it is running, then it can be one min maybe. That said, I'd err on it being longer - say 2-3 mins.

Reply:
so on slow clusters pull image and start > 1 minute ...

Reply:
Countdown begins before the pod is initiated. If the agent isn't connected to pod, then the pod never starts, the timeout takes place. Here's countdown [block](https://github.com/jenkinsci/kubernetes-plugin/blob/35682a458529e42e94c2a905b21f935dfaca7758/src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesLauncher.java#L144)..
Now that I remember, cd projects that are loaded with builds were taking around a min to connect. Maybe I am just too impatient to wait 4 mins for the pod to connect to agents, I'd stop the pipeline thinking somethings wrong.

Reply:
Since you guys vouch for 4 mins, I take you've experienced successful connections that happened past a min or two